Odata-docs/client/v6/basic-crud-operations.md

```diff
@@ -12,6 +12,10 @@ ms.topic: article
 
 **Applies To**: [!INCLUDE[appliesto-odataclient](../../includes/appliesto-odataclient-v6.md)]
 
+## With Codegen/Connected service
```

Comment:
```suggestion
## With Connected Service
```
Also this heading should be at different level than "Request an entity set" or "Request an individual entity"

Reply:
Considering you are introducing a new way of doing things you need a better intro for entire "Basic Crud Operations". It should clearly indicate that there are 2 ways: using ConnService and using POCO.
Are there any benefits or risks for choosing one approach over the other? This should be clearly outlined
In which use cases should one be chosen over the other? (Should also be outlined)